db/seeds.rb

```diff
@@ -25,3 +25,31 @@
   city_name: 'NYC',
   time_zone: 'America/New_York'
 )
+
+Widget.create(
+  name: "Guests",
+  enabled: false,
+  duration_seconds: 20,
+  order: 0
```

Comment:
@mariana0pachon I think you missed these in the renaming from `order` to `position`.